src/index.html

```diff
@@ -5,8 +5,53 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Edu+NSW+ACT+Foundation:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <h1 class="main__title">Take your time</h1>
+      
+      <div class="calendar calendar--month-length_31">
+        <div class="calendar__start-day_mon">
```

Comment:
You should use modifiers to block near blocks class name.
```suggestion
      <div class="calendar calendar--month-length--31 calendar--start-day--mon">
        <div class="">
```
Add modifiers name following BEM naming convention for complex modifiers: block-name--modifier-name--modifier-value (checklist)

Reply:
When i do as you show i see this: 
![image](https://user-images.githubusercontent.com/80990312/180457171-b5a18396-5a5b-4c16-9049-f426235b5a4c.png)


so did a bit different.  if there will be mistakes again, it would be better if I will discuss it on the next QnA with mentor